### PR TITLE
BROKERS: Logging broker #28 #29 #30 #31

### LINKED ISSUE #28: BROKERS: LoggingBroker - LogTrace

### LINKED ISSUE #29: BROKERS: LoggingBroker - LogDebug

### LINKED ISSUE #30: BROKERS: LoggingBroker - LogWarning

### LINKED ISSUE #31: BROKERS: LoggingBroker - LogCritical

### PR DESCRIPTION
**Pull request by SlimAhmad**
Closes #31 
Closes #30 
Closes #29
Closes #28

### CODE REVIEW

File: GitFyle.Core.Api/Brokers/Loggings/LoggingBroker.cs [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    public class LoggingBroker : ILoggingBroker
+    {
+        private readonly ILogger<LoggingBroker> logger;
+
+        public LoggingBroker(ILogger<LoggingBroker> logger) =>
+            this.logger = logger;
+
+        public void LogTrace(string message) =>
+            this.logger.LogTrace(message);
+        public void LogDebug(string message) =>
```
Comment (hassanhabib) [inline GitFyle.Core.Api/Brokers/Loggings/LoggingBroker.cs:19]: Separate two-liners with a new line.
Do the same for the rest please

**Review by hassanhabib — COMMENTED** [review]
One method at a time please

**Comment by SlimAhmad** [comment]
> One method at a time please

Noted thanks

**Comment by cjdutoit** [comment]
> One method at a time please

@SlimAhmad, please remember that GitFyle assigns a score for each issue or PR you complete. If you group them together like this, you will only receive one score for the PR. Separately, you could be awarded a score four times.

**Comment by cjdutoit** [comment]
@SlimAhmad I am going to close this pull request.  Please submit a new pull request for each item.

**Pull request closed without merging**